packages/api/blocks-api/src/blocks/vimeo-video.block.ts

```diff
@@ -0,0 +1,30 @@
+import { IsBoolean, IsOptional, IsString } from "class-validator";
+
+import { BlockData, BlockDataInterface, BlockInput, createBlock, inputToData } from "./block";
+import { BlockField } from "./decorators/field";
+
+class VimeoVideoBlockData extends BlockData {
+    @BlockField({ nullable: true })
+    vimeoIdentifier?: string;
+
+    @BlockField({ nullable: true })
+    autoplay?: boolean;
+}
+
+class VimeoVideoBlockInput extends BlockInput {
+    @IsOptional()
```

Comment:
Please us `@IsUndefinable()` instead

Reply:
Did you forget to push the change?

Reply:
No we talked about it. `The @IsUndefinable()` only exists in `cms-api` and not in `blocks-api`